website/src/components/Chat/ChatConversation.tsx

```diff
@@ -313,6 +317,7 @@ const ChatMessageEntry = memo(function ChatMessageEntry({
                 forceHideCount
                 onClick={handleThumbsDown}
               />
+              <WorkParametersDisplay parameters={workParameters} />
```

Comment:
The params might be null here

Reply:
Since we only execute this code block on assistant messages, it won't be null, but I added a check regardless.